package/harvester-os/files/usr/sbin/harv-install

```diff
@@ -316,6 +316,18 @@ update_grub_settings()
     sed -i "s/search.fs_uuid /search --no-floppy --fs-uuid /" "$GRUB_CFG"
     sed -i -E 's/^(\s*search\b)(.*)root$/\1 --set=root\2/' "$GRUB_CFG"
 
+    # In some customers' hardware and its firmware, we may stuck on the
+    # grub `search` device with the file for about 30 minutes. We add
+    # the following workaround to make grub could search this `grubenv`
+    # file instead of somehow being stuck on `search`.
```

Comment:
```suggestion
    # For some firmware (e.g., Dell BOSS adapter 3022), users may get stuck on the grub file search for about 30 minutes, this can be mitigated by adding the `grubenv` file.
```

Reply:
Done, also update the commit message. Thanks for the reminder!